en/docs/learn/consume-api/manage-subscription/advanced-topics/adding-an-api-subscription-tier-update-workflow.md

```diff
@@ -28,11 +23,11 @@ First, enable the API subscription update workflow for **Approval Workflow Execu
     The subscription update approval workflow executor is now engaged.
 
 
-3.  Sign in to the WSO2 API Developer Portal (`https://<hostname>:<port>/devportal`) and click on **Applications** and select the application which has the subscriptions you wish to modify. Go to the **Subscriptions** tab.
+3.  Sign in to the WSO2 API Developer Portal (`https://<hostname>:<port>/devportal`) and click on **Applications**. Select the application which has the subscriptions you wish to modify. Go to the **Subscriptions** tab from the left navigational menu.
```

Comment:
- Multiple steps have been cramped into one step.
- "Go to the **Subscriptions** tab from the left navigational menu." - Based on the common writing standard that we follow we use images to show the placement of the menu/tab. So this should change to "Click **Subscriptions**" as the image is there to show the user the placement of the tab.